tests/nat/agent/test_base.py

```
@@ -324,8 +324,8 @@ def test_log_tool_response_uses_instance_max_chars(self, caplog):
         # Create a concrete implementation of BaseAgent for testing
         class TestAgent(BaseAgent):
 
-            async def _build_graph(self, state_schema: type) -> CompiledGraph:
-                return Mock(spec=CompiledGraph)
+            async def _build_graph(self, state_schema: type) -> CompiledStateGraph:
```

Ruff (0.12.2)

Unused method argument: `state_schema`

(ARG002)